spec/requests/api/merge_requests_spec.rb

```diff
@@ -114,6 +114,19 @@
     end
   end
 
+  describe "GET /projects/:id/merge_request/:merge_request_id/files" do
+    it "should return the file information of the merge_request" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.